composer.json

```diff
@@ -17,11 +17,8 @@
     "require": {
         "php": "^7.4|^8.0",
         "tipoff/authorization": "^2.5.5",
-        "tipoff/checkout": "^2.4.2",
-        "tipoff/invoices": "^2.0.2",
         "tipoff/locations": "^2.5.4",
-        "tipoff/refunds": "^2.1.0",
```

Comment:
Removal of checkout, invoices and refunds as direct dependencies is intentional.  Refunds will end up with a dependency on payments, but not the other way around.  The checkout and invoice dependencies aren't required.

Reply:
I like that. I also agree with the reverse relationship with refunds.